### ISSUE TITLE
Hard to remove cards

### ISSUE THREAD

**Issue by evajurman** [pokemon-tcg/deck-builder#3]
Don't know what cards in the deck list are related to what ID's, making it difficult to add and remove cards.

Possible solutions:
- Display Card Name next to ID in deck list.
- Click/ Hover to show card in deck list.

**Comment by kristenmills** [comment]
Good point. I kind of want to figure out a good way to add deleting on the card itself. Maybe hover to add a copy or delete a copy.

**Comment by JRJurman** [comment]
Maybe doing the ol' `x` in the corner, like on images or mac virtual desktops... But that doesn't lend itself to group deleting...

**Comment by kristenmills** [comment]
I figure if you want to delete the group, you can do it over in the list. I'll probably do an x and plus on hover of the card, and then also show what card it is in the list.
